tasks.py

```diff
@@ -94,7 +94,7 @@ def install(c):
     print("Installing required python packages from 'requirements.txt'")
 
     # Install required Python packages with PIP
-    c.run('pip3 install -U -r requirements.txt')
+    c.run('pip3 install --no-cache-dir --disable-pip-version-check -U -r requirements.txt')
```

Comment:
Question: Won't `--no-cache-dir` increase the download traffic a lot?

Reply:
Hmm not sure about that. Might be worth checking. My understanding was that it reduced the final docker image size

Reply:
Did not know that - always learning new stuff! Then I think this is the right step 👍